packages/react/src/components/Carousel/Carousel.tsx

```diff
@@ -90,6 +93,8 @@ const Carousel: FC<CarouselProps> & WithWrapperProps = (props: CarouselProps): R
 
   const classes: string = clsx('oxygen-carousel', className);
 
+  const isMobileView: boolean = useMediaQuery((theme: Theme) => theme.breakpoints.down('sm'));
```

Comment:
```suggestion
  const isMobile: boolean = useMediaQuery((theme: Theme) => theme.breakpoints.down('sm'));
```
`isMobile` is used in other compinents. Shall we select a consistent name?